app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -22,7 +24,26 @@ def self.event_monitor_class
 
   def create_project(project)
     connect.create_project_request(project)
-  rescue KubeException => e
```

Comment:
@bzwei Is the recommendation to... 

1. Remove the exception entirely leaving it to the caller to handle?  This I've done in my recent commit

2. Catch the KubeException and  re-raise as an MiqException?